src/GCM.js

```diff
@@ -110,4 +108,3 @@ if (typeof process !== 'undefined' && process.env.NODE_ENV === 'test') {
   GCM.generateGCMPayload = generateGCMPayload;
   GCM.sliceDevices = sliceDevices;
 }
-module.exports = GCM;
```

Comment:
I tried...but somehow exporting the GCM function that takes in arguments causes me to be unable to initialize GCM without needing to pass in args to the ```function GCM(args) {``` that gets exported.